### DISCUSSION TITLE
TabView (Null check operator used on a null value)

### DISCUSSION THREAD

**Discussion by LucasWagner040** [bdlukaa/fluent_ui#606] [Q&A]
Ocorre o erro (`Null check operator used on a null value`) quando entro em alguma tela que tem a `TabView`
O erro acontece apenas em algumas vezes que entro na tela.
O erro corresponde ao `ScrollPosController`
Como posso prosseguir com este erro?

![image](https://user-images.githubusercontent.com/74301870/200861613-79cbb532-f733-4b82-bb11-5892adac0489.png)

**Comment by bdlukaa** [comment]
Qual versão está sendo usada?

**Comment by LucasWagner040** [comment]
![image](https://user-images.githubusercontent.com/74301870/200941186-9a912c68-e10c-468e-9589-e56b703fb2c1.png)

**Reply by bdlukaa** [reply]
Could you provide a reproducible snippet?

**Reply by LucasWagner040** [reply]
![image](https://user-images.githubusercontent.com/74301870/201099740-1c8baf12-6503-4d01-9837-7195e4c48b07.png)

é possível de colocar algum vídeo aqui? para mostrar melhor o erro acontecendo

**Reply by bdlukaa** [reply]
Sim, só fazer o drag-drop do arquivo
